keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt

```diff
@@ -113,7 +113,7 @@ class ArtifactMetadataService(
     )
 
     return retry.executeSuspendFunction {
-      buildService.getArtifactMetadata(commitId = commitId.trim(), buildNumber = buildNumber.trim())
+      buildService.getArtifactMetadata(commitId = commitId.trim(), buildNumber = buildNumber.trim(), completionStatus = CompletionStatus.values().joinToString { it.name })
```

Comment:
I _think_ this works, but have you tested calling igor just to check? (I mean, as opposed to specifying the parameter as a list).

Reply:
Never mind, it works.

Reply:
I have tested it and it is working :) here's an example:
https://igor-prestaging.us-west-2.spinnaker.mgmt.netflix.net:7004/ci/builds?commitId=04dfb52b3d81b6a6c0f744b8e2cbc45f0cb40502&buildNumber=124&completionStatus=INCOMPLETE%2C%20SUCCEEDED%2C%20FAILED%2C%20ABORTED